Makefile

```diff
@@ -17,8 +17,9 @@ GOGET=$(GOCMD) get
 
 .PHONY: gogen
 gogen:
-	@protoc --go_out=plugins=grpc:${GOPATH}/src ./proto/types/*
-	@protoc --go_out=plugins=grpc:${GOPATH}/src ./proto/rpc/*
-	@protoc --go_out=plugins=grpc:${GOPATH}/src ./proto/testing/*
-	@protoc -I. -I./proto/types --go_out=plugins=grpc:${GOPATH}/src ./proto/api/*
-	@protoc -I. --grpc-gateway_out=logtostderr=true:${GOPATH}/src ./proto/api/*
+	@protoc -I. -I${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --go_out=plugins=grpc:${GOPATH}/src ./proto/types/*
```

Comment:
is there any change to api.pb.go after rerun the gogen script?

Reply:
a lot of changes to api.pb.go and api.pb.gw.go,update to this pr?